datastore/longhorn.go

```diff
@@ -437,3 +439,61 @@ func (s *DataStore) ListEngineImages() (map[string]*longhorn.EngineImage, error)
 	}
 	return itemMap, nil
 }
+
+func (s *DataStore) CreateNode(node *longhorn.Node) (*longhorn.Node, error) {
+	return s.lhClient.LonghornV1alpha1().Nodes(s.namespace).Create(node)
+}
+
+// CreateDefaultNode will set default directory to node replica mount path
+func (s *DataStore) CreateDefaultNode(name string) (*longhorn.Node, error) {
+	disks := map[string]types.Disk{}
+	defaultDisk := types.Disk{
+		Path: longhornDirectory,
+	}
+	disks[longhornDirectory] = defaultDisk
+
+	node := &longhorn.Node{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: name,
+		},
+		Spec: types.NodeSpec{
```

Comment:
It's better add a finalizer, so we can handle the deletion (which we won't allow for the user but node does go down (and take manager with it) sometime and we need to react to that).